lib/resque/plugins/disable_job/stats.rb

```diff
@@ -9,25 +9,22 @@ module DisableJob
       # These are methods that inspect the rules
       class Stats
         def self.all_disabled_jobs
-          Hash[Job.disabled_jobs.map { |name| [name, job_disabled_rules(name)] }]
+          Job.disabled_jobs.map { |name| job_disabled_rules(name) }.flatten
         end
 
         def self.job_disabled_rules(name)
-          Resque.redis.hgetall(Rule.new(name).all_rules_key)
+          rules = []
+          Resque.redis.hgetall(Rule.new(name).all_rules_key).each_pair do |digest, arguments|
```

Comment:
you can do `each_with_object([]) do |rules, (digest, arguments)|` to remove the temporary variable

Reply:
Good idea